packages/observe-sequence/observe_sequence_tests.js

```diff
@@ -93,6 +93,37 @@ runOneObserveSequenceTestCase = function (test, sequenceFunc,
              compress(EJSON.stringify(expectedCallbacks, {canonical: true, indent: true})));
 };
 
+// This creates an object subclassed from Array
+const hasProp = {}.hasOwnProperty;
+const extend = function(child, parent) {
+  function ctor() {
+    this.constructor = child;
+  }
+  for(let key in parent) {
+    if(hasProp.call(parent, key)) {
+      child[key] = parent[key];
+      console.log('key: ' + key);
+    }
+  }
+  ctor.prototype = parent.prototype;
+  child.prototype = new ctor();
+  child.__super__ = parent.prototype;
+  return child;
+};
+const ArraySubclass = (function (superClass) {
+  extend(ArraySubclass, superClass);
+
+  function ArraySubclass() {
+    if (arguments.length > 0) {
+      const items = this.slice.call(arguments, 0);
+      this.splice.apply(this, [0, 0].concat(this.slice.call(items)));
+    }
+  }
+
+  return ArraySubclass;
+
+})(Array);
+
```

Comment:
After further consideration, my recommendation wouldn't have worked since `meteor-node-stubs` aren't available from the tool tests, though you should be able to write a similar test in your own app and have it work – assuming you have `meteor-node-stubs` installed.